spanner/api/Spanner.Samples/CreateInstanceWithProcessingUnits.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_create_instance_with_processing_units]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Spanner.Admin.Instance.V1;
+using Google.Cloud.Spanner.Common.V1;
+using Google.LongRunning;
+using Google.Protobuf.WellKnownTypes;
+using System;
+
+public class CreateInstanceWithProcessingUnitsSample
+{
+    public Instance CreateInstanceWithProcessingUnits(string projectId, string instanceId)
+    {
+        // Create the InstanceAdminClient instance.
+        InstanceAdminClient instanceAdminClient = InstanceAdminClient.Create();
+
+        InstanceName instanceName = InstanceName.FromProjectInstance(projectId, instanceId);
+
+        // Initialize request parameters.
+        Instance instance = new Instance
+        {
+            InstanceName = instanceName,
+            ConfigAsInstanceConfigName = InstanceConfigName.FromProjectInstanceConfig(projectId, "regional-us-central1"),
+            DisplayName = "This is a display name.",
+            ProcessingUnits = 500,
+            Labels =
+            {
+                { "cloud_spanner_samples", "true" },
+            }
+        };
+        ProjectName projectName = ProjectName.FromProject(projectId);
+
+        // Make the CreateInstance request.
+        Operation<Instance, CreateInstanceMetadata> response = instanceAdminClient.CreateInstance(projectName, instanceId, instance);
+
+        Console.WriteLine("Waiting for the operation to finish.");
+
+        // Poll until the returned long-running operation is complete.
+        Operation<Instance, CreateInstanceMetadata> completedResponse = response.PollUntilCompleted();
+
+        if (completedResponse.IsFaulted)
+        {
+            Console.WriteLine($"Error while creating instance: {completedResponse.Exception}");
+            throw completedResponse.Exception;
+        }
+
+        Console.WriteLine($"Instance created successfully.");
+
+        // GetInstance to get the processing units using a field mask.
```

Comment:
We usually would have two separate samples for this, one for creating the instance, and one for obtaining the processing units. It's fine if docs have been designed with this as a single sample, just saying that it is going to be slightly different to the rest of the samples in that sense.

Reply:
Yeah I totally understand. We unfortunately will only have one sample for this feature so we decided to put both the create and get in one place :(